src/SWP/Bundle/CoreBundle/Controller/ContainerController.php

```diff
@@ -14,6 +14,7 @@
 
 namespace SWP\Bundle\CoreBundle\Controller;
```

Comment:
but if I understood correctly the CoreBundle dependency will be optional, It will be required only if you would want to make use of Controllers in our project and if someone will want to make use of it he/she will need to implement its own logic or the similar one we have in core bundle to handle controllers responses properly